app/users/resources.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, User
+from app.lib.response_util import buildOkResponse
+
+
+def returnUser(user):
```

Comment:
To represent a private method, we typically add a leading `_`. Could you change this to `_returnUser`?